src/main/java/Controllers/ReviewManager.java

```diff
@@ -0,0 +1,61 @@
+package Controllers;
+
+import Entities.Review;
+import UseCaseClasses.InformationSaver;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+public class ReviewManager {
+
+    private HashMap<String, Review> reviews;
```

Comment:
if you need to access a list of reviews, you may need to create a new class called ReviewList, to avoid directly creating Reviews (entities) and thus avoid clean architecture violations